### PR TITLE
Add the instruction of running the benchmark tests

### LINKED ISSUE #280: Add benchmark for rclnodejs
We want to create the PnP (Power and Performance) tests for rclnodejs to compare with its counterparts - [rclpy](https://github.com/ros2/rclpy) and [rclcpp](https://github.com/ros2/rclcpp), and we want to collect data from the following aspects:

- CPU usage
- Execution time
- Memory cost

The whole task has been divided into three sub issues: #277, #278 and #279.

### PR DESCRIPTION
**Pull request by minggangw**
Fix #280

### CODE REVIEW

File: benchmark/README.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+* [Linux](https://github.com/ros2/ros2/wiki/Linux-Install-Binary)
+* [macOS](https://github.com/ros2/ros2/wiki/OSX-Install-Binary)
+* [Windows](https://github.com/ros2/ros2/wiki/Windows-Install-Binary)
+
+2.Install Node.js
+
+Download from https://nodejs.org/en/
+
+3.Clone the repo `git clone https://github.com/RobotWebTools/rclnodejs.git` and `npm install`
```
Comment (qiuzhong) [inline benchmark/README.md]: Actually, there is one more step before `npm install`: update the sub modules. I would suggest to just say `rclnodejs` installed.

File: benchmark/README.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+1.Install ROS 2 from binary
+
+You can download the latest binary package of ROS2 from [here](http://ci.ros2.org/view/packaging/) and follow the instructions to setup the environment.
+
+* [Linux](https://github.com/ros2/ros2/wiki/Linux-Install-Binary)
+* [macOS](https://github.com/ros2/ros2/wiki/OSX-Install-Binary)
+* [Windows](https://github.com/ros2/ros2/wiki/Windows-Install-Binary)
+
+2.Install Node.js
```
Comment (qiuzhong) [inline benchmark/README.md:13]: Better to indicate the `Node.js` version, suggest to use latest `Node.js` LTS version always.

**Comment by minggangw** [comment]
@Honry @hanyia @qiuzhong the document used to instruct the running of PnP test has been added, combine with #292 #294 #298 please review them all, thanks!

File: benchmark/README.md (hunk 2 above)
Comment (minggangw) [reply benchmark/README.md:13]: I will add to point out the Node.js version as latest LTS

File: benchmark/README.md (hunk 1 above)
Comment (minggangw) [reply benchmark/README.md]: We can guide users to read https://github.com/RobotWebTools/rclnodejs#get-code

**Comment by minggangw** [comment]
@Honry @hanyia @qiuzhong Please review this pr again. If there is no comment, I will merge this pr and #292 #294 #298 to get the PnP ready!

**Comment by Honry** [comment]
LGTM

**Review by Honry — APPROVED** [review]

**Comment by qiuzhong** [comment]
The C++ and Python PRs are good for me.

**Comment by hanyia** [comment]
Please merge, then let us do the initial testing, thanks.

**Comment by minggangw** [comment]
@Honry @hanyia @qiuzhong All the PnP related PRs have been merged.

**Pull request merged**